src/examples/libpmemobj/hashmap/hashmap_atomic.c

```diff
@@ -58,11 +59,12 @@ struct entry_args {
 	PMEMoid value;
 };
 
+POBJ_LIST_HEAD(entries_head, struct entry);
 struct buckets {
 	/* number of buckets */
 	size_t nbuckets;
 	/* array of lists */
-	POBJ_LIST_HEAD(entries_head, struct entry) bucket[];
```

Comment:
Is this macro illegal inside the struct definition? If that's the case than you should fix the macro, not the invocation.